force-app/main/default/classes/TreeViewRecords.cls

```diff
@@ -0,0 +1,37 @@
+public with sharing class TreeViewRecords {
+    @AuraEnabled(cacheable=true)
+    public static List<TreeViewWrapper> getRecordsTreeView(String objectAPITreeView, String fieldsAPI){
+        
+        String query = '';
+        String[] fields;
+        
+        if(!String.isEmpty(fieldsAPI)) {
+            query = 'SELECT Name, Id, CreatedDate, ' + fieldsAPI + ' FROM ' + objectAPITreeView + ' ORDER BY CreatedDate ASC LIMIT 25';
+            fields = fieldsAPI.split(',');
+        }
+        else {
+            query = 'SELECT Name, Id, CreatedDate FROM ' + objectAPITreeView + ' ORDER BY CreatedDate ASC LIMIT 25';
```

Comment:
The same here, please use String.escapeSingleQuotes for both objectAPIDataTable and fieldsAPI input parameters.